src/Libraries/CoreNodeModels/Properties/Resources.en-US.resx

```diff
@@ -394,7 +394,7 @@
     <value>List</value>
   </data>
   <data name="PortDataOperandToolTip" xml:space="preserve">
-    <value>operand</value>
+    <value>Boolean #</value>
```

Comment:
which node is this used for? and what do we mean by `Boolean #`

Reply:
In nodes `And` and `Or` when you hovered over the first 2 inputs you got as description: operand, when you did it over the next input you got Boolean # so did this to keep consistency
![s3](https://user-images.githubusercontent.com/25138291/92121731-8526c300-edf2-11ea-96e1-e5759ff8721e.png)

Reply:
so if you hover over `bool0` in the new version is the tooltip just `Boolean #` ?

Reply:
"Boolean #" + index